index.md

```diff
@@ -12,7 +12,7 @@ title: Home
 
 <div class="container">
   <div class="image-container">
-    <img src="/assets/photo.jpg" alt="moi" style="width: 200px">
+    <img src="./assets/photo.jpg" alt="moi" style="width: 200px">
```

Comment:
@Badr-MOUFAD just to be sure I understand, the ./ is not needed here since the index is a the root of the website, no?

Reply:
Yes, you are right.
In this case, both are fine, either with a dot or without, see for example

https://github.com/mathurinm/mathurinm.github.io/blob/131d11550018d33e5a49f061a37af9f45ef08ae9/index.md?plain=1#L46


I added a dot to preview the render locally, otherwise ``/asserts/photot.jpg`` is treated as an absolute path.

Reply:
If you run `bundle exec jekyll serve -o` at the root of the repo it should work fine (at least it works for me)